cypress.json

```diff
@@ -1 +1 @@
-{}
+{ "projectId": "1fuai2" }
```

Comment:
Just out of curiosity: was there a specific reason this was added?

Reply:
Hey Johannes!

Every time I push the changes, the CI pipeline pointed out that Cypress has failed and couldn't figure out what was wrong as I only edited the bug. then tried to fix the cypress thing and that's why I added it and CI run successfully.

Reply:
But I've edited it out now, run again and it behaved well with my last commit.

Reply:
One of the Cypress tests is a bit flaky unfortunately. Sometimes it helps to just re-run the CI jobs. But no idea if that was the problem